src/components/calcite-card/calcite-card.tsx

```diff
@@ -76,8 +77,8 @@ export class CalciteCard {
   render(): VNode {
     const dir = getElementDir(this.el);
     return (
-      <Host dir={dir}>
-        <div class="calcite-card-container">
+      <Host>
+        <div class={{ ["calcite-card-container"]: true, [CSS_UTILITY.rtl]: dir === "rtl" }}>
```

Comment:
I don't think you need the square brackets around `"calcite-card-container"` here?